tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6637,9 +6661,17 @@ Status ConvertResize(OpConverterParams* params) {
 
   // Set layer parameters.
   layer->setResizeMode(resize_mode);
-  layer->setOutputDimensions(output_dimensions);
   layer->setAlignCorners(align_corners);
 
+  // Set output shape.
+  if (params->use_implicit_batch || HasStaticShape(inputs_dims)) {
+    // If all shape dims are known at build time, pass them as a parameter.
```

Comment:
Let's revise this comment:
If the input shape is known at build time, pass the static output shape to the resize layer as expected output dimensions.